src/main/content/_assets/css/404page.css

```diff
@@ -0,0 +1,364 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+ #page_container {
+    position: relative;
+    background-image: -webkit-gradient(linear, left bottom, left top, color-stop(9%, #515b7d), to(#1f203e));
+    background-image: -webkit-linear-gradient(bottom, #515b7d 9%, #1f203e 100%);
+    background-image: -o-linear-gradient(bottom, #515b7d 9%, #1f203e 100%);
+    background-image: linear-gradient(0deg, #515b7d 9%, #1f203e 100%);
+    padding-top: 227px;
+    overflow: hidden;
+}
+
+#error_message {
+    font-family: BunueloLight;
+    font-size: 40px;
+    color: white;
+    text-align: center;
+    width: 559px;
+    margin-right: auto;
+    margin-left: auto;
+    margin-bottom: 270px;
+}
+
+#crashed_ship_container {
+    position: relative;
+    margin-left: 50%;
+    -webkit-transform: translateX(-50%);
+        -ms-transform: translateX(-50%);
+            transform: translateX(-50%);
+    width: 915px;
+}
+
+#crashed_ship {
+    width: 916px;
+    height: 413px;
+}
+
+#shiplight_top, #shiplight_mid, #shiplight_bot,
+#right_planet, #left_planet,
+#smoke1, #smoke2, #smoke3,
+#smoke4, #smoke5, #smoke6,
+#smstar_br, #smstar_bl, #smstar_c,
+#medstar_r, #medstar_l, #bigstar,
+#star_cluster1, #star_cluster2, #star_cluster3 {
+    position: absolute;
+}
+
+#shiplight_top, #shiplight_mid, #shiplight_bot {
+    -webkit-animation: shiplight_blink 10s infinite;
+            animation: shiplight_blink 10s infinite;
+}
+
+#shiplight_top {
+    top: 15px;
+    left: 287px;
+}
+
+#shiplight_mid {
+    top: 72px;
+    left: 318px;
+    -webkit-animation-delay: -9.5s;
+            animation-delay: -9.5s;
+}
+
+#shiplight_bot {
+    top: 119px;
+    left: 377px;
+    width: 13px;
+    -webkit-animation-delay: -9s;
+            animation-delay: -9s;
+}
+
+#smoke1, #smoke2, #smoke3,
```

Comment:
Same for the smokes, having a `smoke` class to apply common styling across them.